csharp-basic-topics/DetectKeyPressInCSharp/Tests/ConsoleAppTests.cs

```diff
@@ -0,0 +1,50 @@
+using DetectKeyPressInCSharp;
+using Moq;
+using NUnit.Framework;
+
+namespace Tests
+{ 
+    [TestFixture]
+    public class ConsoleAppTests
+    {
+        private Mock<IConsoleService> mockConsoleService;
+        private ConsoleApp app;
+
+        [SetUp]
+        public void Setup()
+        {
+            mockConsoleService = new Mock<IConsoleService>();
+            app = new ConsoleApp(mockConsoleService.Object);
+        }
+
+        [Test]
```

Comment:
Also, test names should be following Given_When_Then. @mohitkumar342

Reply:
DOne